examples/brute_force.py

```diff
@@ -43,7 +43,23 @@
 
 flags.DEFINE_list(
     "actions",
-    [],
+    [
+        "-simplifycfg",
+        "-reg2mem",
+        "-early-cse-memssa",
+        "-gvn-hoist",
+        "-gvn",
+        "-instsimplify",
+        "-instcombine",
+        "-jump-threading",
+        "-loop-extract",
+        "-loop-reduce",
+        "-loop-versioning",
+        "-newgvn",
+        "-mem2reg",
+        "-sroa",
+        "-structurizecfg",
+    ],
```

Comment:
I would prefer to leave the default value as an empty list. This script is not specific to LLVM, and atm an empty list will auto-select all of the actions available in the environment